app/views/requests/_request_row.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= request_row.comments %>
   </td>
   <td>
-    <%= request_row&.request_type&.titleize %>
+    <%= request_row&.request_type&.first&.capitalize %>
```

Comment:
Can we extract this into a method on `request`? It's a bit obscure. If you call it something like `request_type_label` with a comment it'd be more obvious.